CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 All notable changes to this project will be documented in this file.
 
```

Comment:
The behavior of the application is changing. This is why i like it to be a  feature / minor release